dotnet3.5/S3/UploadObjectExample/UploadObject/UploadObject.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. 
+// SPDX-License-Identifier:  Apache - 2.0
+
+using Amazon;
+using Amazon.S3;
+using Amazon.S3.Model;
+using System;
+using System.Threading.Tasks;
+
+namespace UploadObject
+{
+    // The following example uploads an object to an Amazon Simple Storage
+    // Service (Amazon S3) bucket in two different ways. First, it uploads
+    // an existing file from the supposed file path and then it creates an
```

Comment:
Since we are uploading two different objects, why not a description like:

This code example contains two methods to upload an object to an Amazon ... The UploadObjectFromFileAsync method uploads an existing file using the specified file path. The UploadObjectFromContentAsync method creates ...

Reply:
I've rewritten the description mention the two included methods by name.